codelists/views/version.py

```diff
@@ -14,7 +16,7 @@ def version(request, clv):
     code_to_term = None
     parent_map = None
     tree_tables = None
-    if clv.coding_system_id in ["bnf", "ctv3", "icd10", "snomedct"]:
+    if issubclass(clv.coding_system.__class__, BuilderCompatibleCodingSystem):
```

Comment:
```suggestion
    if isinstance(clv.coding_system, BuilderCompatibleCodingSystem):
```

I think checking `isinstance` is sufficient, more idiomatic, and easier to read than accessing `__class__`, here.

Reply:
I can't believe I missed the bleeding obvious there :facepalm:  thanks!